test/views/ViewSpec.scala

```diff
@@ -33,14 +33,16 @@ class ViewSpec extends WordSpec with MustMatchers with ViewTemplates with ViewMa
 
   implicit protected def htmlBodyOf(html: Html): Document = Jsoup.parse(html.toString())
 
-  private val messagesApi: MessagesApi = instanceOf[MessagesApi]
+  private val messagesApi: MessagesApi = ViewSpec.realMessagesApi
```

Comment:
??

Reply:
We could not have separate injector for each test case with each own object tree - safe of memory and GC pressure